v4/source/users.yaml

```diff
@@ -128,6 +128,36 @@
         '401':
           $ref: '#/responses/Unauthorized'
 
+  /users/names:
```

Comment:
Oh, so we do. I specifically looked for that and I thought it was name. I'll fix that